lib/action_cable/railtie.rb

```diff
@@ -0,0 +1,19 @@
+module ActionCable
+  class Railtie < Rails::Railtie
+    config.action_cable = ActiveSupport::OrderedOptions.new
+
+    initializer "action_cable.logger" do
+      ActiveSupport.on_load(:action_cable) { self.logger ||= ::Rails.logger }
+    end
+
+    initializer "action_cable.set_configs" do |app|
+      options = app.config.action_cable
+
+      options.allowed_request_origins ||= "http://localhost:3000" if ::Rails.env.development?
```

Comment:
@jeremy The `config.action_cable.logger = …` currently works because the logger is set on line 6 to the Rails.logger. If this in the same structure that I saw in other rails gems. Is it ok like that?